safe/impact_function/impact_function.py

```diff
@@ -1021,9 +1021,16 @@ def run(self):
             suggestion_heading = m.Heading(
                 tr('Suggestion'), **SUGGESTION_STYLE)
             suggestion = tr(
-                'Check in your .qgis2/python/plugins directory that you do '
-                'not have a processing folder. You should use the Processing '
-                'plugin provided by QGIS.')
+                'InaSAFE depends on the QGIS Processing plugin. This is a '
+                'core plugin that ships with QGIS. It is also possible to '
+                'install the processing plugin from the QGIS Plugin Manager, '
```

Comment:
The plugin is not shipped anymore on plugins.qgis.org. You can't download it. You can only enable/disable it in the plugin manager.